vocabs/languages-iso639-1.ttl

```diff
@@ -0,0 +1,1488 @@
+@prefix rdf:          <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:         <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:      <http://purl.org/dc/terms/> .
+@prefix skos:         <http://www.w3.org/2004/02/skos/core#> .
+@prefix voaf:         <http://purl.org/vocommons/voaf#> .
+@prefix schema:       <https://schema.org/> .
+@prefix vann:         <http://purl.org/vocab/vann/> .
+@prefix loclang:      <http://id.loc.gov/vocabulary/iso639-1/> .
+@prefix lang:         <http://w3id.org/drk/vocabs/languages-iso639-1#> .
+
+lang:
```

Comment:
Interesting to see this. Looks well done. Just a question from the maintenance/efficiency perspective, how did you create this? All manually? And, hasn't something comparable and reusable existed already?